packages/hadron-document/package.json

```diff
@@ -12,29 +12,45 @@
     "type": "git",
     "url": "https://github.com/mongodb-js/compass.git"
   },
+  "engines": {
+    "node": ">=14.17.5"
+  },
   "license": "SSPL",
   "keywords": [
     "mongodb-js"
   ],
-  "main": "index.js",
-  "types": "./index.d.ts",
+  "main": "lib/index.js",
   "exports": {
-    "webpack": "./src/index.js",
-    "require": "./index.js"
+    "webpack": "./src/index.ts",
+    "require": "./lib/index.js",
```

Comment:
Total super nit, but maybe we can align this with most packages in the monorepo and make it `dist` here (and then we will need less special ignore files in this workspace)